config/topic-browser-test.yaml

```
@@ -0,0 +1,34 @@
+    umh_topic: "umh.v1.*"  # Match all UMH topics
+    kafka_topic: "umh.messages"
+    broker_address: "localhost:9092"
+    consumer_group: "benthos_kafka_test"
+
+pipeline:
+  processors:
+    - topic_browser: {}
+
+output:
+  stdout:
+    codec: lines 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)